static/build/miyoo/app/lang/gl.lang

```diff
@@ -0,0 +1,161 @@
+{
+	"lang": "Galego (Galician)",
+	"0": "Experto",
+	"1": "Favoritos",
+	"2": "Sistemas",
+	"3": "ARCADE",
+	"4": "NEOGEO",
+	"5": "FC",
+	"6": "SFC",
+	"7": "GB",
+	"8": "GBA",
+	"9": "MD",
+	"10": "GPSP",
+	"11": "SMS",
+	"12": "PS",
+	"13": "PCE",
+	"14": "OUTROS",
+	"15": "Axustes",
+	"16": "INFO. XOGO",
+	"17": "INFO. CONSOLA",
+	"18": "Recentes",
+	"19": "ARQUIVO",
+	"20": "ADC INFO",
+	"21": "EncDec Test",
+	"22": "Mapeo botóns",
+	"23": "Idioma",
+	"24": "Volume",
+	"25": "Volume música",
+	"26": "Brillo",
+	"27": "Actualizar ROMs",
+	"28": "Restauración de fábrica",
+	"29": "Modo flasheo",
+	"30": "Información dispositivo",
+	"31": "Proba hardware",
+	"32": "Info ADC",
+	"33": "Estado:",
+	"34": "Retardo:",
+	"35": "Taxa OK:",
+	"36": "Taxa fallos:",
+	"37": "DETIDO",
+	"38": "CORRENDO",
+	"39": "ADC:",
+	"40": "Cargando",
+	"41": "OTG",
+	"42": "Desconectado",
+	"43": "Erro",
+	"44": "Favoritos",
+	"45": "Cancelar",
+	"46": "OK",
+	"47": "segundos",
+	"48": "en",
+	"49": "Lista recentes",
+	"50": "Limpar",
+	"51": "Abrir",
+	"52": "Eliminar",
+	"53": "Refrescar",
+	"54": "Refrescar ROMs",
+	"55": "Engadir como favorito",
+	"56": "Actualizar",
+	"57": "¿Actualizar firmware?",
```

Comment:
~~¿Actualizar firmware?~~
**Actualizar firmware?**

There is no inverted question mark in Galician.